src/editor/editor.js

```diff
@@ -275,13 +275,24 @@ export default class Editor {
 	execute( ...args ) {
 		this.commands.execute( ...args );
 	}
+
+	/**
+	 * Creates and initializes a new editor instance.
+	 *
+	 * @static
+	 * @method module:core/editor/editor~Editor.create
+	 * @param {Object} config The editor config. You can find the list of config options in
+	 * {@link module:core/editor/editorconfig~EditorConfig}.
+	 * @returns {Promise} Promise resolved once editor is ready.
+	 * @returns {module:core/editor/editor~Editor} return.editor The editor instance.
+	 */
```

Comment:
Do we need this doc if the method does not exist anymore?

Reply:
It exists in subclasses (but has own docs there) but also `editor#create()` method is linked in some places in this package (where it makes sense IMHO) and linking to subclasses (creators) `create()` method doesn't make sense in context of `core` package. So I suppose it makes sense to have it here.

Reply:
> method is linked in some places in this package (where it makes sense IMHO)

I do not think so. There is no such method and there will be no such method. It is also not requested for all subclasses to create such a method, so I think you should update docs which liked to it.

Reply:
And can't we have a `@returns {Promise.<module:core/editor/editor~Editor>} Promise that resolves to the editor instance once the editor is ready.` if we go with documenting this method? I don't get these multiple `returns` statements honestly...

Reply:
It's hard to describe the initialization process when there is one step (`create()` method) which is missing from the base class but may be present (or at the current state always is) in the whole process. So from the point of view of the `Editor` class there is no `create()` method, but in real cases it is the core part / entry point of the whole process. For me it feels like something is not in the right place.

Anyway, I corrected the docs removing links to `create()` method trying to make them readable and easy to understand (see https://github.com/ckeditor/ckeditor5-core/pull/159/commits/b2b65f47d87d1a0f5c7194a34ac0c38917f9f3a9).

---

Btw. there are two places in the docs which I'm not sure how to handle:

1. https://ckeditor.com/docs/ckeditor5/latest/builds/guides/integration/frameworks/angular.html#editor-required - where should we link in such cases?
1. https://ckeditor.com/docs/ckeditor5/latest/framework/guides/architecture/core-editor-architecture.html#editor-classes - should `create()` just be skipped here as it is not present in the base class?

Reply:
Let's continue this in https://github.com/ckeditor/ckeditor5/issues/1487.

Reply:
😭 The part you mentioned was actually a leftover of the old `create()` docs. I removed it and now we are left with docs proposed by @Reinmar in https://github.com/ckeditor/ckeditor5/issues/1487#issuecomment-458601552 - there are no `@param`s nor `@returns` which aligns with your first point:

> not document `@param {Object} config` here because different editors can have different arguments. For instance the classic editor has `static create( sourceElementOrData, config )`. We mustn't enforce any order or type of arguments here – it's up to the developer what it looks like.